### PR TITLE
[Extensions] Kubectl Plugin for Kanvas Snapshot

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

### CODE REVIEW

File: collections/_extensions/kubectl-kanvas-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+layout: item
+item-type: extension
+name: Kubectl Plugin for Kanvas Snapshot 
+kind: GitOps
+userName: Meshery Authors
+type: Configuration
+compatibility: 
+  - kubernetes
+  - kanvas
+# extensionId: 2d54f372-10e5-4c4e-8d23-b9c35c68ce96
```
Comment (miacycle) [inline collections/_extensions/kubectl-kanvas-snapshot.md:11]: Is this not needed?

**Review by miacycle — APPROVED** [review]

**Review by suttonskate — COMMENTED** [review]
I see that the avatar URL for users was removed. Is that not necessary? I see it in the other extensions....

**Review by jameshorton2337 — APPROVED** [review]
lgtm

**Comment by jameshorton2337** [comment]
Oh, wait. I was expecting to see a coming soon flag or ribbon on top of this. Do we have one?

**Pull request merged**